documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -139,6 +139,24 @@ you will need the new `hibernate-search-elasticsearch` jar.
 ----
 ====
 
+If your Elasticsearch instance is hosted on Amazon Web Services,
```

Comment:
We need to clarify this, in particular what you mean by "hosted"

I talked with David Pilato about this feature and it turns out the Elasticsearch is not involved with the AWS-managed services, so they generally recommend running it "unmanaged" on your own hardware (even though possibly run on EC2) or via their own cloud offering - which is also hosted on AWS.

So I think:
 - "running it on AWS" is not the right premise
 - also referring to "and if your instance is not public" -> nobody will do that right?

What about "If you use Amazon's proprietary `IAM` authentication mechanism to access Elasticsearch..." ?